File: app/app/src/main/java/com/example/bankmanagement/view/full_screen_image/FullScreenImage.java
package com.example.bankmanagement.view.full_screen_image;


import android.annotation.SuppressLint;
import android.app.Activity;
import android.graphics.Bitmap;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

import com.example.bankmanagement.R;

public class FullScreenImage  extends Activity {


    @SuppressLint("NewApi")



    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.layout_full);

        Bundle extras = getIntent().getExtras();
        Bitmap bmp = (Bitmap) extras.getParcelable("imagebitmap");

        ImageView imgDisplay;
        Button btnClose;


        imgDisplay = (ImageView) findViewById(R.id.imgDisplay);
        btnClose = (Button) findViewById(R.id.btnClose);


        btnClose.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                FullScreenImage.this.finish();
            }
        });


        imgDisplay.setImageBitmap(bmp );

    }


}

File: app/app/src/main/java/com/example/bankmanagement/view/full_screen_image/FullScreenImage.kt
package com.example.bankmanagement.view.full_screen_image

import android.app.Activity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.ImageView
import com.example.bankmanagement.R
import com.example.bankmanagement.utils.Utils.Companion.getBitmapFromURL
import kotlinx.coroutines.Dispatchers
import kotlinx.coroutines.GlobalScope
import kotlinx.coroutines.launch
import kotlinx.coroutines.withContext

class FullScreenImage : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.layout_full)

        val extras = intent.extras
        val link = extras!!.getString("imageLink")


        val imgDisplay: ImageView = findViewById<View>(R.id.imgDisplay) as ImageView
        val btnClose: Button = findViewById<View>(R.id.btnClose) as Button


        btnClose.setOnClickListener { finish() }
        GlobalScope.launch(Dispatchers.IO) {

            val bmp = getBitmapFromURL(link)
            withContext(Dispatchers.Main) {
                imgDisplay.setImageBitmap(bmp)

            }
        }
    }
}